src/stylesheets/base/_animation-mixins.scss

```diff
@@ -0,0 +1,41 @@
+@mixin animation($name, $duration: 0.6s, $easing: ease-in-out) {
+  -webkit-animation: $name $duration;
+  -moz-animation: $name $duration;
+  animation: $name $duration;
+}
+
+@mixin keyframes($name) {
+  @keyframes #{$name} {
+    @content;
+  }
+}
+
+@mixin shake($rotate1, $rotate2, $rotate3, $scale, $tx1, $tx2, $tx3) {
```

Comment:
Produces keyframes for pills being translated with the translate3d property.

Reply:
Is there a reason your using translate3d? It doesn't look like you're using a "Z" value other than "0" so I don't think you need to differientate between `shake2d` and `shake` mixins. Just have a single shake mixin using rotate/scale/translate internally